torch/csrc/inductor/aoti_package/model_package_loader.cpp

```diff
@@ -10,13 +10,12 @@
 #include <nlohmann/json.hpp>
 #include <fstream>
 #include <iostream>
-
 #ifndef _WIN32
 #include <sys/stat.h>
-#else
+#endif
+
 #include <filesystem>
 namespace fs = std::filesystem;
```

Comment:
We can't use std::filesystem due to https://github.com/pytorch/pytorch/issues/133437. I know it's annoying, but until you resolve that linking issue, we have to still with the current way.